addOns/automation/src/main/resources/org/zaproxy/addon/automation/resources/addOns-min.yaml

```diff
@@ -0,0 +1,5 @@
+  - type: addOns                       # Add-on management
+    parameters:
+      updateAddOns: true               # Update any add-ons that have new versions
+    install:                           # A list on non standard add-ons to install from the ZAP Marketplace
+    uninstall:                         # A list of standard add-ons to uninstall
```

Comment:
Well, in the future they could be different, and right now the names are formed in the abstract class by appending the -min.yaml and -max.yaml suffixes - changing this would complicate the code a bit more so swings and roundabouts?